### DISCUSSION TITLE
Vanilla Subs Additions or Reworks

### DISCUSSION THREAD

**Discussion by JamesTCrenshaw** [FakeFishGames/Barotrauma#4742] [Ideas and suggestions]
The current roster of submarines that come with the game are lacking in amount, as well as balancing; barring two of them.

The two most balanced subs, in my, and some others' opinions are: the _Humpback_ and the _Dugong_. While these subs can sometimes be infuriating, or lacking in some aspect or another, they happen to be the most fun to play vessels on the roster. The reason these two subs are so much fun to play, (despite most players, including myself having complaints about the subs for some reason or another) is boiled down to three main factors: The amount and frequency of systemic issues of the sub, the capability or lack thereof of the sub, and the survivability of a total calamitous event.

The systemic issues of the sub are issues that are intentionally left unsolved, or partially fixed. These problems are not intended to be problems for problems' sake, but rather to be issues that the players must work around, solve, or occasionally encounter and must deal with. These issues cannot immediately or primarily destroy the ship or its crew, but rather initiate a cascading series of events that occur in which the entire crew must fulfill each of their roles to save the ship.

Example: The _Humpback_ has an issue with power systems as well as a 'blind spot' in which the dorsal turret is unable to see or shoot enemies that are eating into the hull directly above command. This circumstance brings two issues that occur that ensure everyone has a job to do in a short amount of time: The enemy will claw their way into command, and begin injuring or incapacitating crew. The hull breach associated with the enemy entering the command room floods much of the ship, and separates the armory, and backup battery array from much of the crew. Some of the crew may have escaped to the armory, and are stranded, due to the fact that water, pressure, and enemies are directly below them.

This series of events has ensured that each job has to do their job, and quickly, to revive the ship. The engineer has to rewire the doors so they can be manually operated, to control the flooding. The engineer also has to reach the batteries and junction boxes to keep the ship operating and removing water, as well as having to deal with the resultant power losses associated with all the pumps activating at the same time. The mechanic has to fix the pumps, as well as assist in hull repairs and flooding control. The medic has to heal those who are injured or incapacitated, and must quickly grab all the medical equipment he can before med-bay floods. The security officer has to kill the intruder, as well as potentially fighting off additional threats around the ship. The captain has to get to navigation, and get the ship into a safe location, or get back to a station to fix the ship. Everyone is accounted for, and everyone is participating. Some of the most memorable and fun events I have observed hosting games was because the _Humpback_'s innate issues led to these situations in which the entire crew had to hurry and fix the vessel.

This comes to capability, or even the slight lack of it. Both the _Dugong_ and the _Humpback_ are fully capable of doing most if not all the missions provided on the campaign. The other ships are sometimes too long, or too girthy to proceed in the map, or lack the weaponry to defeat the larger monsters should they attack from an undefended angle. The _Dugong_ has a good capability/sacrifice set, by being able to fight most creatures, but being unable to fabricate at all. This leads to more planning, and wreck salvaging than would normally occur, which allows for more gameplay opportunities for such a small crew size, without overly complicating the game with having to deal with fabrication. The same goes for the _Humpback_, which lacks the power capacity to sustain all the systems at once, unless using backup power. This gives more gameplay situations in which players must pick and choose what systems they want to use or repair, and what systems they may want to disable to save the ship from itself.

Lastly, survivability. Both these vessels are easily sunk, however they can survive even the most brutal and total destruction of the vessel, with minimal crew required to repair it. The _Dugong_ can be repaired with a minimum of one crew member, while the _Humpback_ can be repaired with a minimum of two. This survivability of the two vessels, allows fantastic gameplay that is fun to watch and play, as they tell the classical story of surviving against all odds. This does not mean that it is easy to repair the vessels, as it certainly takes a significant amount of work, but the satisfaction that the players get when they finally dock the ship that survived a nuclear blast that killed most the crew and ripped half the hull away, is priceless. Even the spectators will walk away from the high stress gameplay they watched in which all were invested, feel satisfied and happy with the gameplay. This is the result that most games strive to achieve, but few can.

All of these things work in harmony to produce a masterpiece work from a gameplay perspective. If you can rework the other ships, or add additional ships that replicate these experiences, players will be significantly happier with the vanilla experience, and turn less to heavily modded subs that innately drive the fun out of the game. (It seems most players misplace dissatisfaction with the vanilla subs, with the problems the subs have. They try to make the 'perfect' sub, that inevitably leave the crew with nothing to do, leaving players dissatisfied and thinking that Barotrauma is the problem, and lacks content, rather than the design of the ship itself being the problem. The ships absolutely cannot be designed to fly themselves. They must have problems, to keep the game fun.

**Comment by mLuby** [comment]
I agree with your analysis: recovering from failure is half the fun.

Is there a suggestion in here?
